node/nodejs_intro/README.md

```diff
@@ -69,55 +102,78 @@ Let's take a look at the official Chrome V8 website: [https://developers.google.
+<details>
+<summary>
+    Answer
+  </summary>
+  Javascript is a "client-side" language (client-side = web application), and before Node.js, we did not have a tool to allow Javascript to talk to server-side applications like databases. Now, we are able to run servers in Node.js, which is also able to interpret Javascript, so we can bridge the gap between Javascript and server-side tech!
+</details>
+<p>&nbsp;</p>
+<details>
+<summary>
+    Important Note
+  </summary>
+  😊 You're doing great! 😊
+</details>
+<p>&nbsp;</p>
 
+# Objective 3: The Event Loop
 
-# 4. The Event Loop
+You might have heard that JavaScript is `single-threaded` , this means it can only do one task at a time, and must wait for that task to finish before picking up another one. You may have also heard that it's [`asynchronous`](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Asynchronous/Concepts), meaning while it waits on a longer task to finish, it can pick up a different task. Well, hold on, that doesn't make sense, how could it be both?! Good question!
 
-You've probably heard that JavaScript is single-threaded, but you've also heard that it's asynchronous. This is probably when you start to say, "that doesn't make sense, how could it be both?". Good question! Answer: JS is single-threaded, the asynchronous behavior is not part of the JS language itself, but is actually built on top of the core JS language in the browser (or other programming environment) and is accessed through the browser APIs.
+Answer: JS itself is 🧵 `single-threaded` 🧵, and it's asynchronous behavior is built on top of the core JS language in the browser (or other programming environment) and is accessed through the browser APIs. [EDIT]
 Take a look at the following picture:
 
-![eventLoopImage](https://i.imgur.com/rnQEY7o.png)
+![eventLoopImage](assets/eventloop.png)
+*Event Loop via [link](https://i.imgur.com/rnQEY7o.png)*
 
-I know it looks pretty weird, but let's break down the different parts of the picture and see if we can get it to make more sense. For now, disregard the heap part.
+Let's break down the different parts of the picture and see if we can get it to make more sense. For now, disregard the heap part.
 
-The outer box is the Google Chrome that we know and love. For this example that is our programming environment.
+- The outer box is the Google Chrome that we know and love. For this example that is our programming environment.
+![Event Loop Chrome](assets/eventloopchrome.png)
 
-The next box we see is the JS box. This is where our code runs. The call stack is where we are in the code. We can only push and pop things onto our stack. This represents the single thread provided.  
+- The next box we see is the JS box. This is where our code runs. The call stack is where we are in the code. We can only push and pop things onto our stack. This represents the 🧵 `single-thread` 🧵 provided.
```

Comment:
try <kbd>JS box</kbd> or `JS Box` for keywords/phrases to help distinguish them as key words or terms. 

```
<kbd>JS box</kbd>
`JS Box`
````

Reply:
Love it